### PR TITLE
German translation improvements

### PR DESCRIPTION
**Pull request by rschattauer**

### CODE REVIEW

File: shared/src/commonMain/composeResources/values-ie/strings.xml [hunk 1]
```diff
@@ -46,4 +46,6 @@
     <string name="empty_text">Gan aon data i láthair</string>
     <string name="no_bookmarks">Gan aon leabharmharcanna leagtha</string>
     <string name="privacy_policy">Beartas Príobháideachta</string>
+    <string name="bookmarks_past">Caite</string>
+    <string name="bookmarks_upcoming">Le teacht</string>
```
Comment (rschattauer) [inline shared/src/commonMain/composeResources/values-ie/strings.xml:50]: no idea if these are correct, feel free to change them

File: shared/src/commonMain/composeResources/values-fr/strings.xml [hunk 2]
```diff
@@ -44,4 +44,6 @@
     <string name="no_paired_watch">Pas de montre appairée</string>
     <string name="no_bookmarks">Aucun favori défini</string>
     <string name="privacy_policy">Politique de confidentialité</string>
+    <string name="bookmarks_past">Passé</string>
+    <string name="bookmarks_upcoming">À venir</string>
```
Comment (rschattauer) [inline shared/src/commonMain/composeResources/values-fr/strings.xml:48]: no idea if these are correct, feel free to change them

File: androidApp/src/main/java/dev/johnoreilly/confetti/bookmarks/BookmarksView.kt [hunk 3]
```diff
@@ -92,9 +94,9 @@ private fun BookmarksContent(
     }
 }
 
-private enum class BookmarksTab(val title: String) {
-    Past(title = "Past"),
-    Upcoming(title = "Upcoming")
+private enum class BookmarksTab(val title: @Composable () -> String) {
```
Comment (rschattauer) [inline androidApp/src/main/java/dev/johnoreilly/confetti/bookmarks/BookmarksView.kt:97]: Either like this or returning StringResource type

**Comment by joreilly** [comment]
Thanks for that!

File: shared/src/commonMain/composeResources/values-fr/strings.xml (hunk 2 above)
Comment (joreilly) [reply shared/src/commonMain/composeResources/values-fr/strings.xml:48]: @martinbonnin wdyt? We can follow up with another PR if they do need to be changed

File: shared/src/commonMain/composeResources/values-de/strings.xml [hunk 4]
```diff
@@ -1,18 +1,18 @@
 <resources>
     <string name="schedule">Zeitplan</string>
-    <string name="sessions">Sitzungen</string>
-    <string name="speakers">Redner</string>
+    <string name="sessions">Vorträge</string>
+    <string name="speakers">Sprecher</string>
     <string name="venue">Veranstaltungsort</string>
     <string name="search">Suchen</string>
     <string name="bookmarks">Lesezeichen</string>
-    <string name="rooms">Zimmer</string>
```
Comment (ashdavies) [inline shared/src/commonMain/composeResources/values-de/strings.xml:8]: 😂

**Pull request merged**